docs-client/src/containers/MethodPage/DebugPage.tsx

```diff
@@ -328,7 +353,7 @@ const DebugPage: React.FunctionComponent<Props> = ({
         `${window.location.port ? `:${window.location.port}` : ''}`;
 
       const httpMethod = method.httpMethod;
-      let uri;
+      let baseUri;
```

Comment:
```suggestion
      let mappedPath;
```
because we use the term for this purpose? 😉